py/kubeflow/testing/run_e2e_workflow.py

```diff
@@ -167,6 +167,12 @@ def run(args, file_handler): # pylint: disable=too-many-statements,too-many-bran
   base_branch_name = os.getenv("PULL_BASE_REF")
   pull_base_sha = os.getenv("PULL_BASE_SHA")
 
+  if args.cloud_provider == "aws":
```

Comment:
I was thinking about if this is necessary to add another lay, because it's easy for developers to see the mappings between cloud_provider and "aws" / "gcp" / "ibm"